src/lightning_app/components/multi_node/base.py

```diff
@@ -17,7 +17,7 @@ def __init__(
         *work_args: Any,
         **work_kwargs: Any,
     ) -> None:
-        """This component enables performing distributed multi-node multi-device training.
+        """This component enables performing distributed multi-node multi- device training.
```

Comment:
I do not think that these kinds of changes are correct...

Reply:
Yes, indeed. I used pre-commit to run auto-formatting, but looks like something wrong with my environment, since it's not what I had expected. I'll check it.